content/docs/operating/configuration.md

```diff
@@ -396,6 +396,9 @@ region: <string>
 # Named AWS profile used to connect to the API.
 [ profile: <string> ]
 
+# AWS Role ARN, an alternative to using AWS API keys.
+[ role_arn: <arn:aws:iam::account-id:role/role-name> ]
```

Comment:
The part enclosed in the carrot brackets is the type of the field, not an example string. I think it's well enough defined by AWS that we don't need a further example here, I'd just use a plain `<string>` type here.

Reply:
Done.